app/forms/import_duplicate_form.rb

```diff
@@ -36,6 +44,11 @@ def apply_changes_options
     can_keep_both? ? %w[apply discard keep_both] : %w[apply discard]
   end
 
+  def imms_record?
```

Comment:
```suggestion
  def can_apply?
```

This matches the naming convention of `can_keep_both?` and keeps the method name generic (this class doesn’t need to know about the Imms API specifically), making it easier to expand in the future if we needed it.

Should we also update `apply_changes_options` to not return `apply` if `can_apply?` is `false`? That way the view could be more generic and in the case of there being only one option (in this case `discard`) display the content specific to the one option.

Reply:
Good idea